tools/d2g/d2g.go

```diff
@@ -324,6 +325,14 @@ func setSupersederURL(dwPayload *dockerworker.DockerWorkerPayload, gwPayload *ge
 	gwPayload.SupersederURL = dwPayload.SupersederURL
 }
 
+func setOSGroups(dwPayload *dockerworker.DockerWorkerPayload, gwPayload *genericworker.GenericWorkerPayload) {
+	if dwPayload.Capabilities.Devices.KVM {
+		// task user needs to be in the kvm and libvirt groups for KVM to work
+		// https://help.ubuntu.com/community/KVM/Installation
+		gwPayload.OSGroups = append(gwPayload.OSGroups, "kvm", "libvirt")
```

Comment:
You do not need libvirt to use KVM. Libvirt is just an orchestrator allowing you to manage virtual machines from different backends via XML files.

Reply:
Thanks Bastien 👍🏻 I'll go ahead and remove then. So looking over at https://github.com/taskcluster/community-tc-config/commit/5a7ebdcf319fb6ac8d85932ddfe40ca0ad27eac7, some of these packages may not be needed on the worker? Or would you suggest keeping these installed in case the user may want to use libvirt?

Reply:
They definitely shouldn't be necessary for getting a VM to run.

I don't know who's using those images, maybe ask them if they need libvirt? I'd personally remove them and add them back if anyone actually needs them and finds it cumbersome to install them during their CI workflow. But I don't know what the stance on that is for community images.